libs/editor/WordPressEditor/build.gradle

```diff
@@ -1,9 +1,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
This sounds good to me - mentioning the gradle wrapper reminded me that the minimum gradle wrapper version for Gradle plugin 3.0 is `4.1`, and the `utils` and `login` libs were only on `4.0.1`. I bumped all the wrappers to `4.1` in 7dedc74 so that we can safely push to the utils/networking/login subtrees.

I've generally been ignoring the editor project since we can't push to the subtree anymore (all the Aztec work is mixed in) - it makes sense to update it though, and I agree we should drop `publishNonDefault` if that's not causing Bintray any problems (I should hope not).

(Once the visual/legacy editors are removed from WPAndroid, I'd like to do one final cleanup and push to the subtree (excluding Aztec changes) so we can retire that repo in a working state.)

What do you say we merge this and I'll review your branch as a separate PR?

Reply:
@hypest I did some test Bintray releases and confirmed `publishNonDefault` has no effect anymore - dropped it in b271ce6 as suggested 👍

Reply:
Thanks for the extra changes @aforcier ! 

> What do you say we merge this and I'll review your branch as a separate PR?

Perfectly fine with me! I'll prepare a PR with the editor's example app changes (you have covered the rest of the changes here anyway) after we merge this one.